easybuild/easyconfigs/h/hhsuite/hhsuite-3.3.0-foss-2020b.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# Based on STAR-2.7.7a-GCC-10.2.0.eb 
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'hhsuite'
```

Comment:
```suggestion
name = 'HH-suite'
```

Reply:
Sure, no problems. I changed that and the filename as well. I also done some tidy up, I hope that looks better now. Thanks for your help.